cmd/syncthing/main.go

```diff
@@ -892,6 +892,10 @@ func setupGUI(mainSvc *suture.Supervisor, cfg *config.Wrapper, m *model.Model, a
 		return
 	}
 
+	if guiCfg.InsecureAdminAccess {
+		l.Warnln("Insecure admin access is enabled.")
```

Comment:
This is a pain in the ass for people who want to proxy behind NGINX or some sort where SSL and authentication is terminated by the webserver. Maybe this could be checkboxed to remove this warning after one occurence?